plugins/inputs/docker/docker.go

```diff
@@ -219,6 +230,21 @@ func gatherContainerStats(
 	gatherBlockIOMetrics(stat, acc, tags, now)
 }
 
+func calculateCPUPercent(stat *docker.Stats) float64 {
+	var (
```

Comment:
this `var` stanza is a little odd, just stylistically I would remove the `var()` part and change to:

```go
var cpuPercent float64
cpuDelta := float64(stat.CPUStats.CPUUsage.TotalUsage) - float64(stat.PreCPUStats.CPUUsage.TotalUsage)
systemDelta := float64(stat.CPUStats.SystemCPUUsage) - float64(stat.PreCPUStats.SystemCPUUsage)
```